spec/api_spec.rb

```diff
@@ -1,20 +1,27 @@
 require 'spec_helper'
 require 'orchestrate'
+require 'timecop'
+require 'time'
 
 describe HRLH::API do
   let(:database) { double('database') }
   let(:client) { double('client') }
   let(:collection) { double('collection') }
-  let(:interviewer_record_in_db) { double(value: {"name" => 'Jichao', "email" => 'jichao@thoughtworks.com'}, key: key) }
-  let(:interviewer) { {name: 'Jichao', email: 'jichao@thoughtworks.com'} }
+  let(:interviewer_record_in_db) { double(value: {'name' => 'Jichao', 'email' => 'jichao@thoughtworks.com', 'language'=>'ruby', 'work_from'=>'2011'}, key: key) }
+  let(:interviewer) { {name: 'Jichao', email: 'jichao@thoughtworks.com', language: 'ruby', experience: 4} }
```

Comment:
Line is too long. [107/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.